astrokat/observe_main.py

```diff
@@ -98,7 +98,7 @@ def observe(session, ref_antenna, target_info, **kwargs):
 
     # simple way to get telescope to slew to target
     if "slewonly" in kwargs:
-        return session.track(target, duration=0.0, announce=False, slew_only=True)
+        return session.track(target, duration=0.0, announce=False)
```

Comment:
With the proposed changes in katcorelib it seems like the 'slewonly' kwarg wouldn't do anything unless it altered the structure of the target mapping being passed to 'track'?

Reply:
@ludwigschwardt I think that the changes in [391](https://github.com/ska-sa/katcorelib/pull/391) can reasonably replace what was done in [113](https://github.com/ska-sa/astrokat/pull/113). This "slewonly" check would only need to ensure that it specifically passes track() a target dict with only an 'ants' key. Then there shouldn't be any impact on the rest of the observation setup or execution as far as I can tell.